.env.example

```diff
@@ -0,0 +1,28 @@
+# You'll need to set the following in either your environment or in a .env
+# file in the root of the project
+#
+# The values in this file are purely examples to better help understand what it
+# is you need to add
+
+PORT=8000
```

Comment:
No problem, I have added a comment and changed the name to: WASTE_PERMITS_APP_PORT